Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -5,11 +5,17 @@
 #include <functional>
 #include <sstream>
 #include <assert.h>
+#include <NativeEngine.h>
 
 namespace Babylon::Polyfills::Internal
 {
     static constexpr auto JS_CONSTRUCTOR_NAME = "NativeCanvas";
 
+    struct NativeCanvas::Impl
+    {
+        std::unique_ptr<Babylon::TextureData> m_textureData;
```

Comment:
For structs with public members, I think this should be `TextureData` instead of `m_textureData`.

Reply:
sounds good. i see this vary in different places. But am happy to start using capitalization throughout.